src/main/java/mate/academy/model/ShoppingCart.java

```diff
@@ -0,0 +1,66 @@
+package mate.academy.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.MapsId;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "shopping_carts")
+public class ShoppingCart {
+    @Id
+    private Long id;
+    @OneToMany
+    @JoinColumn(name = "cart_id")
+    private List<Ticket> tickets;
+    @OneToOne(fetch = FetchType.LAZY)
+    @MapsId
+    private User user;
```

Comment:
```suggestion
    @MapsId
    @JoinColumn(name = "shopping_cart_id")
    private User user;
```

lets name our column shopping_cart_id